data/download_datasets.py

```
@@ -0,0 +1,177 @@
+    if not output_file.exists():
+        download_url(url, output_file)
+        print(f"✓ Downloaded to {output_file}")
+    else:
+        print(f"✓ Already exists: {output_file}")
+    
+    # Extract
+    extract_dir = libritts_dir / "LibriTTS"
+    if not extract_dir.exists():
+        print("Extracting archive...")
+        with tarfile.open(output_file, "r:gz") as tar:
+            tar.extractall(path=libritts_dir)
```

Ruff (0.14.13)

Uses of `tarfile.extractall()`

(S202)